### PR TITLE
 remove set map to address button

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by jbraswell** [comment]
You also need to remove the `setMapToAddress` function in server_admin_javascript.js. Then you need to remove javascript references to the button.

Lines 2378 and 2397.

**Comment by pjaudiomv** [comment]
good call

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ CHANGELIST
 - Latitude and longitude input boxes are now read only instead of disabled, allowing users to select and copy the calculated values.
 - Fixed an issue where static files in the client_interface/html/ endpoint were not being included.
 - The German translations for the administration UI were updated.
+- Removed "Set Map to Address button" in the single meeting editor as not needed anymore and messed up workflow.
```
Comment (jbraswell) [inline README.md]: The "Set Map to Address" button in the single meeting editor was removed, as it no longer provides values to the workflow.

Comment (pjaudiomv) [reply README.md]: profresh

**Pull request merged**